Software 0.30.2 release notes (#1247)
astronomer/docs: software/release-notes.md

0.30 is the latest stable and long-term support (LTS) version of Astronomer Software. To upgrade to 0.30, see [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, see [Release and lifecycle policies](release-lifecycle-policy.md). To read release notes specifically for the Astro CLI, see [Astro CLI release notes](cli-release-notes.md).

## 0.30.2

> Release date: September 23, 2022

Reviewer: was released on Sep 22

> Release date: September 23, 2022

Author: ```suggestion
Release date: September 22, 2022
```

### Additional improvements

> - You can now set a `securityContext` for Fluentd Pods and containers using the [Fluentd Helm chart](https://github.com/astronomer/astronomer/blob/master/charts/fluentd/values.yaml).

Reviewer 2: ```suggestion
- You can now use the [Fluentd Helm chart](https://github.com/astronomer/astronomer/blob/master/charts/fluentd/values.yaml) to set a `securityContext` for Fluentd Pods and containers.
```

- Improved the startup time for the platform NATS server.

> 

Reviewer: The main bug fix that went in to this release was https://github.com/astronomer/issues/issues/5046. Dont we want to add a note about that? cc: @cmarteepants

> 

Author: ```suggestion
- You can now configure external containers in the `astronomer.houston.config` section of the Astronomer Helm chart.
```

### Bug fixes

> - Fixed several CVEs as a result of updating images for system components. 

Reviewer 2: I assume that you don't want to define _CVE_? Typically, you define acronyms on first use.

> ### Bug fixes

Author: ```suggestion

### Bug fixes
```
